.github/workflows/main.yml

```diff
@@ -9,8 +9,9 @@ jobs:
     name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }}
     runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: false
       matrix:
-        version: ['1.0', '1.1', '1.2', '1.3']
+        version: ['1.0', '1.1', '1.2', '1.3', '1.4']
```

Comment:
nice I can see an update on the versions ;)

Reply:
They passed too ;)